### PR TITLE
IP 13

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
I wasn't sure if this was a truly a letter or not, so I wasn't sure about my use of a postscript, especially because the postscript is dated before the letter is.

File: unabridged-works-volume-4/original/13-compulsion-in-religion.adoc [hunk 1]
```diff
@@ -3,35 +3,44 @@
...
+[.numbered]
+_Query 4._ Whether God hath given any man or men power, by laws and penalties,
 to compel men to worship him contrary to the belief of their
 hearts and the full persuasions of their souls?
 Is it not written, that whatsoever is not of faith is sin?
 And hath God given any men power and authority to compel others to sin?
 Surely the end of magistracy is to restrain men from sin, and not to compel men to sin.
 
+[.numbered]
 Query 5. Whether under the law of Moses,
```
Comment (jaredh159) [inline unabridged-works-volume-4/original/13-compulsion-in-religion.adoc:37]: Looks like all of the other `Query X.` you italicized, but not this one.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Looks great, just one little change. 👍

**Review by jaredh159 — APPROVED** [review]

**Comment by jaredh159** [comment]
thanks!

**Pull request merged**